src/components/dashboard/Claim.js

```diff
@@ -41,7 +42,7 @@ const Claim = props => {
   const gdstore = GDStore.useStore()
 
   const { entitlement } = gdstore.get('account')
-  const isCitizen = gdstore.get('isLoggedInCitizen')
+  const isCitizen = gdstore.get('isLoggedInCitizen') || true
```

Comment:
Why the isCitizen variable is true when gdstore.get('isLoggedInCitizen') is not?

Reply:
ohhps! good catch, that was for testing